nixio/block.py

```diff
@@ -83,11 +83,19 @@ def create_multi_tag(self, name="", type_="", positions=0,
             return self.multi_tags[id]
 
         util.check_entity_name_and_type(name, type_)
-        util.check_entity_input(positions)
         multi_tags = self._h5group.open_group("multi_tags")
         if name in multi_tags:
             raise exceptions.DuplicateName("create_multi_tag")
-        mtag = MultiTag._create_new(self, multi_tags, name, type_, positions)
+        if not isinstance(positions, DataArray):
+            da_name = "{}-positions".format(name)
+            positions = self.create_data_array(da_name, "{}-positions".format(type_),
```

Comment:
Might be good to catch any DA creation exceptions here, specifically any exception that will be thrown if the `positions` data is invalid.  We could then re-raise it from here with a more appropriate message that mentions the MultiTag positions array explicitly.